WordPress/Classes/ViewRelated/In-App Update/BlockingUpdateViewModel.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+struct BlockingUpdateViewModel {
+    let appName: String
+    let detailsString: String
+    let releaseNotes: [String.SubSequence]
+    let onUpdateTapped: () -> Void
+
+    init(appStoreInfo: AppStoreInfo, onUpdateTapped: @escaping () -> Void) {
+        self.appName = appStoreInfo.trackName
+        self.detailsString = makeDetailsString(from: appStoreInfo)
+        self.releaseNotes = appStoreInfo.releaseNotes.split(whereSeparator: \.isNewline)
+        self.onUpdateTapped = onUpdateTapped
+    }
+}
+
+private func makeDetailsString(from appStoreInfo: AppStoreInfo) -> String {
+    let latestVersion = appStoreInfo.version
+    let fileSizeBytes = ByteCountFormatter.string(fromByteCount: Int64(appStoreInfo.fileSizeBytes) ?? 0, countStyle: .file)
```

Comment:
Not sure if we want to show it. 
Is it accurate? The device-specific builds are usually smaller.